languages.yaml

```diff
@@ -1,3 +1,32 @@
+ABNF:
+  type: data
+  ascii: |
+    {0}         ______ 
+    {0}        |      |
+    {0}        |  /\  |
+    {0}>>--+-->| /--\ |-->+--->>
+    {0}    |   |______|   |
+    {0}    |    ______    |
+    {0}    |   |  _   |   |
+    {0}    |   | |_)  |   |
+    {0}    +-->| |_)  |-->+
+    {0}    |   |______|   |
+    {0}    |    ______    |
+    {0}    |   |      |   |
+    {0}    |   | |\ | |   |
+    {0}    +-->| | \| |-->+
+    {0}    |   |______|   |
+    {0}    |    ______    |
+    {0}    |   |   _  |   |
+    {0}    |   |  |_  |   |
+    {0}    +-->|  |   |-->+
+    {0}        |______|   
```

Comment:
```suggestion
    {0}          ______
    {0}         |      |
    {0}         |  /\  |
    {0}>>---+-->| /--\ |-->+--->>
    {0}     |   |______|   |
    {0}     |    ______    |
    {0}     |   |  __  |   |
    {0}     |   | |__) |   |
    {0}     +-->| |__) |-->+
    {0}     |   |______|   |
    {0}     |    ______    |
    {0}     |   |      |   |
    {0}     |   | |\ | |   |
    {0}     +-->| | \| |-->+
    {0}     |   |______|   |
    {0}     |    ______    |
    {0}     |   |  ___ |   |
    {0}     |   | |___ |   |
    {0}     +-->| |    |-->+
    {0}         |______| 
```
I suggest increasing the width of letters `B` and `F` to match the other letters.

![image](https://user-images.githubusercontent.com/13710835/194727171-f78ed9b9-3b4e-4bb7-9df4-baf24d130be7.png)